GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/RAWFILE.H

```diff
@@ -48,17 +48,10 @@
 
 // #include	"win.h"
 
-#ifdef _UNIX
```

Comment:
Add a comment to the top of this file highlighting that the winapi file handling around CreateFileA was removed.